### PR TITLE
Bump The Lounge to v2.7.1

### PR DESCRIPTION
**Pull request by astorije**
Disabling `jsonlint` for now because:

- https://twitter.com/astorije/status/965453311204823040
- https://github.com/zaach/jsonlint/pull/105
- https://github.com/zaach/jsonlint/pull/104
- https://github.com/zaach/jsonlint/issues/103
- https://www.npmjs.com/package/nomnom

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2018 Jérémie Astori
+Copyright (c) 2015 Jérémie Astori
```
Comment (andreineculau) [inline LICENSE:3]: I guess this was not intentional

Comment (astorije) [reply LICENSE:3]: Thanks for noticing, @andreineculau!
It was intentional actually, and the reason is the same as mentioned here: https://github.com/thelounge/deb-lounge/pull/16

Comment (andreineculau) [reply LICENSE:3]: You say there that you believe in bumping the copyright years when the project is updated. You have already done so in January 2018 for this project. Here you reversed the change. Anyways, i was nitpicking while looking at the actual jsonlint disabling

Comment (astorije) [reply LICENSE:3]: Right, I meant specifically xPaw's comment:

> My understanding is that copyright years shouldn't be updated. https://stackoverflow.com/a/2391555/2200891

So most likely I was wrong, and corrected my wrong here :)

Yeah, disabling `jsonlint` isn't great, but npm...

**Comment by andreineculau** [comment]
Aha :)
Re:jsonlint yeah, always great to find old stuff that always worked just stop working...

--
http://andreineculau.com

**Pull request merged**